web/samples_index/web/styles.scss

```diff
@@ -432,7 +432,7 @@ a {
       background-color: black;
       opacity: 0.2;
       cursor: pointer;
-      transition: 40ms ease;
+      transition: 200ms ease;
```

Comment:
Is this part of the fix or just something that should be changed?

Reply:
no, not really :) I increased it to give some smoothness to the animation